_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: Trino in 2020 - An amazing year in review
+author: Martin Traverso, Manfred Moser, Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+**Wow!** If you would have to sum up what happened in the last that year in this
+great community, that would be it. It is truly awe-inspiring to be part of this
+incredible journey of Trino. Oh, yeah on that note. Our community and project
+[chose the new name Trino]({% post_url 2020-12-27-announcing-trino %}), to be
+able to continue to innovate and develop freely as a community of peers. Presto®
+and Presto SQL are a thing of the past.
+
+Now that is out of the way, let’s dive right in, and see we all our community
+members across the globe have created with us!
+
+<!--more-->
+
+## By the numbers
+
+Even the size and growth of [our community on Slack](/slack.html) is impressive:
+
+* Started 2020 with ~1600 members and 280 weekly active
+* Over 3200 members
+* 560 members active weekly
+
+The innovation and change of [the source code on GitHub]({{
+site.github_repo_url}}) is a result of the hard work of the community:
+
+* Over 4000 commits merge
+* More than 2800 pull requests
+* [23 releases](/docs/current/release.html#releases-2020), nearly every two
+  weeks basically!
+
+As you can see, much of the excitement around the name change has quickly
+increased the number of stars we have on Github. While some of this certainly
+stems from an initial buzz around a shiny new name, we also believe that this
+name change has brought clarity to the community. Trino is not one and the
+same as Presto®, but in fact a much newer and improved version. We expect to see
+this trajectory play out as our predecessor fades away.
+
+And if you have not done so already, make sure to [star the
+repository]({{site.github_repo_url}}) and [join us on slack](/slack.html).
+
+## Community resources and events
+
+Beyond the raw code and helping each other, the community was also busy
+collaborating on other media.
+
+The book [Presto: The Definitive Guide](/presto-the-definitive-guide.html) was
+completed by Matt, Manfred and Martin and published by O'Reilly. And over 5000
```

Comment:
i know everyone knows who Matt, Manfred and Martin are, but maybe make names into links?
especially for the first occurrence of a person

Reply:
Sure